tests/app/schema_loader/test_schema_loader.py

```diff
@@ -1,29 +1,54 @@
+import os
+
+from httmock import urlmatch, HTTMock, response
+from werkzeug.exceptions import NotFound
+
 from app.utilities.schema import load_schema_from_params
 
 from tests.app.app_context_test_case import AppContextTestCase
 
 
 class SchemaLoaderTest(AppContextTestCase):
 
+    @staticmethod
+    @urlmatch(netloc=r'(.*\.)?')
+    def register_mock(url, request): # pylint: disable=unused-argument
```

Comment:
I *think* pylint will ignore vars beginning with '_' so you should be able to replace 'request' with '_' and drop the disable comment.